### PR TITLE
Feature/subscribe to new study created gdev 366

### PR DESCRIPTION
**Pull request by MoritzHahn1337**
Title for squash and merge:
```
Subscribe to new-study-created message (GDEV-366, GDEV-284)
```

Description for squash and merge:
```
Subscribed to new-study-created message
Extract file information from message
Write file information to upload-controller database
```

### CODE REVIEW

**Comment by MoritzHahn1337** [comment]
template_repo needs to adapt the copyright year, then static files won't fail

File: upload_controller_service/pubsub/subscribe.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        for file in files
+    ]
+
+    handle_new_study(study_files=study_files, config=config)
+
+
+def subscribe_file_staged(config: Config = CONFIG, run_forever: bool = True) -> None:
+    """
+    Runs a subscribing process for the "file_staged_for_download topic"
+    """
```
Comment (KerstenBreuer) [inline upload_controller_service/pubsub/subscribe.py]:
```suggestion
def subscribe_new_study_created(config: Config = CONFIG, run_forever: bool = True) -> None:
    """
    Runs a subscribing process for the "new_study_created" topic
    """
```

**Review by KerstenBreuer — CHANGES_REQUESTED** [review]
Looks very good. Just tests are missing.

**Comment by MoritzHahn1337** [comment]
Tests will be added in a future PR

**Review by KerstenBreuer — APPROVED** [review]

**Pull request merged**